src/js/va-letters/containers/DownloadLetters.jsx

```diff
@@ -1,18 +1,44 @@
 import React from 'react';
 import { connect } from 'react-redux';
 
+// TODO: style this properly once design is settled
 class DownloadLetters extends React.Component {
   render() {
+    const destination = this.props.destination || {};
+    const letterItems = (this.props.letters || []).map((letter) => {
+      return (
+        <li key={letter.letterType}>
+          <a href="#">{letter.letterName}</a>
+        </li>
+      );
+    });
     return (
-      <div>Placeholder for download letters</div>
+      <span>
+        <div className="letters-form-panel">
+          <p>The address on file for you with VA Compensation and Pension is:</p>
+          <span>{destination.fullName}</span><br/>
+          <span>{destination.addressLine1}, {destination.addressLine2} {destination.addressLine3}</span><br/>
+          <span>{destination.city}, {destination.state} {destination.zipCode}</span>
+        </div>
+        <div>
+          <p>The following letters are available for download:</p>
+          <ul>
+            {letterItems}
+          </ul>
+        </div>
+      </span>
     );
   }
 }
 
 function mapStateToProps(state) {
   const userState = state.user;
+  const letterState = state.letters;
   return {
-    profile: userState.profile
+    profile: userState.profile,
+    letters: letterState.letters,
+    destination: letterState.destination,
+    available: letterState.available
```

Comment:
My understanding of what "available" means is here: https://github.com/department-of-veterans-affairs/vets.gov-team/issues/2671
but according to that logic, this should be part of the letter-type response (i.e., it's a property of each individual letter, not a property of the letter list.

In that case, I guess a value of `false` would tell us we need to put up a notice that no letters are available due to a system being down or whatever

Tangentially, we need to sort out how the EVSS API provides information about availability and eligibility when it comes to individual letters, i.e.,
- if a user is eligible for a letter, but a backend service is down, so we should show the letter and gray it out with some explanation of what's going on
- if a user is not eligible for a letter, we won't show it in the UI

I don't like that they are confounding the concepts of availability and eligibility...will make sure we discuss this with them either via slack or at Thursday's meeting.